b3lb/rest/b3lb/lb.py

```diff
@@ -130,18 +147,19 @@ def get_node_params_by_lowest_workload(cluster_group):
         return None
 
 
-def get_slide_body_for_post(slide):
-    slide_path = "rest/slides/{}".format(slide)
-    try:
-        slide_file = open(slide_path, "rb")
-        body = '<modules><module name="presentation"><document name="{}">{}</document></module></modules>'.format(slide, base64.b64encode(slide_file.read()).decode())
-        slide_file.close()
-    except FileNotFoundError:
-        body = None
+def get_slide_body_for_post(asset, secret):
```

Comment:
The `secret` should not be required (see also URL stuff above), the `asset` already contains the `tenant`.